models.py

```diff
@@ -119,3 +119,4 @@ class Blacklist(Base):
     __tablename__ = 'blacklist'
     name = Column(String, primary_key=True, nullable=False)
     added_by = Column(Integer, ForeignKey('users.id'),nullable=False)
+    added_at = Column(DateTime)
```

Comment:
I'd rather see added_at be the following: 
```python
added_at = Column(DateTime, nullable=False, default=datetime.utcnow())
```

Its more explicit when looking at the code what the default value is.